tests/test_dmlscheduler.py

```diff
@@ -90,3 +94,12 @@ def test_dmlscheduler_cron():
         initial_weights = output.results['weights']
         assert type(initial_weights) == list
         assert type(initial_weights[0]) == np.ndarray
+
+# def test_dmlscheduler_cannot_schedule_transform_and_split_jobs(mnist_filepath):
```

Comment:
If this has been resolved, you can go ahead and remove it?

Reply:
It's not resolved really -if we add the transform function back in, this test will still be useful to show us that multiprocessing can't handle transform_and_split.

Reply:
Wait why? If this is the Pickle Error, we can just convert the lambda function to a regular function.

Reply:
And regardless, I don't think we'll be passing transform functions back and forth anyway per Georgy's reasoning.

Reply:
Ok, removed